app/3.0/docs/tools/polymer-cli-commands.md

```diff
@@ -163,21 +196,23 @@ See also:
 
 ## polymer install {#install}
 
-Installs Bower dependencies. Running `polymer install` is equivalent to running `bower install`.
+Installs dependencies.
 
-The `--variants` flag allows you to install dependency variants. See the documentation on [managing dependencies for hybrid elements](/{{{polymer_version_dir}}}/docs/devguide/hybrid-elements#dependency-variants) for more information.
+If the --npm flag is given, running `polymer install` is equivalent to running `npm install`:
```

Comment:
Uh... we need to double check this. No one's worked on `polymer install` for npm, we don't have variants for npm, and I'm inclined to just deprecate the command.

Reply:
https://github.com/Polymer/tools/blob/master/packages/cli/README.md#polymer-install---variants

https://github.com/Polymer/tools/pull/303

@justinfagnani Looks like it has been done. I will update the docs to include the info about `polymer install` unless there's some reason I shouldn't.

cc @bicknellr

Reply:
Yes, `polymer install --npm` just runs `npm install` for you and all the other flags *should* do nothing.